content/en/tracing/guide/ingestion_control_page.md

```diff
@@ -0,0 +1,112 @@
+
+Infrastructure
+: Hosts, containers and functions on which the service is running.
+
+Service status
+: Shows `Limited Resource` when some spans are dropped due to the datadog agent reaching CPU or RAM limits, `Legacy Setup` when some spans are ingested through the legacy [App Analytics mechanism][5], and `OK` the rest of the time.
+
+Filter the page by environment, configuration and status to view services for which you need to take an action. To reduce the global ingestion volume, sort the table by the `Downstream Bytes/s` column to view services responsible for the largest share of your ingestion.
+
+**Note:** The table is powered by the [usage metrics][8] `datadog.estimated_usage.apm.ingested_spans` and `datadog.estimated_usage.apm.ingested_bytes`. These metrics are tagged by `service`, `env` and `ingestion_reason`.
+
+### Traffic breakdown
+
+The Traffic Breakdown column breaks down the destination of all traces originating from the service. It gives you an estimation of the share of traffic that is ingested vs dropped, and for which reasons.
+
+{{< img src="tracing/trace_indexing_and_ingestion/service_traffic_breakdown.png" style="width:100%;" alt="traffic breakdown of trace ingestion" >}}
+
+The breakdown is composed of the following parts:
+
+- **Complete traces ingested** (blue): The percentage of traces that have been ingested by Datadog.
+- **Complete traces not retained** (gray): The percentage of traces that have intentionally not been forwarded to Datadog by the agent or the tracer. This can happen for one of two reasons depending on your configuration:
+
+    1. By default, the [agent distributes an ingestion rate][6] to services depending of services traffic.
+    2. When the service is manually [configured][7] to ingest a certain percentage of traces at the tracing library level
+
+- **Complete traces dropped by the tracer rate limiter** (orange): When you choose to manually set the service ingestion rate as a percentage with rules, a rate limiter is set to 100 traces per second by default is automatically enabled. See the [rate limiter][7] documentation to manually configure this rate.
+
+- **Traces dropped due to the agent CPU or RAM limit** (red): This mechanism may drop spans and create incomplete traces. To remediate to the issue, upscale the agent infra CPU/memory allocation.
+
+## Service Ingestion Summary
+
+Click on any service row to view the Service Ingestion Summary, a detailed view providing actionable insights on the ingestion configuration of the service.
+
+
+{{< img src="tracing/trace_indexing_and_ingestion/service_ingestion_summary.png" style="width:100%;" alt="Service Ingestion Summary" >}}
```

Comment:
nit: this image will be outdated very soon due to the button label change we're doing now

Reply:
Thanks for the headsup. Please lmk when the change is done

Reply:
It's been merged. You should be able to see it.